storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Test/TestAPI.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using CommandLine;
+using Dotnet.Storm.Adapter.Channels;
+using Dotnet.Storm.Adapter.Components;
+using Dotnet.Storm.Adapter.Logging;
+using Dotnet.Storm.Adapter.Messaging;
+using Dotnet.Storm.Adapter.Serializers;
+using log4net;
+using log4net.Config;
+using log4net.Core;
+
+namespace Dotnet.Storm.Adapter.Test
+{
+    public static class TestAPI
```

Comment:
Good to add documentation comments /// to public methods

Reply:
I would add this when it is finalized. For now, it seems there are changing along the time.